WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -96,16 +120,18 @@ open class WordPressOrgXMLRPCApi: NSObject {
     }
 
     deinit {
-        _sessionManager?.session.finishTasksAndInvalidate()
-        _uploadSessionManager?.session.finishTasksAndInvalidate()
+        for session in [urlSession, uploadURLSession, sessionManager.session, uploadSessionManager.session] {
+            session.finishTasksAndInvalidate()
```

Comment:
I didn't look into these calls too much—the new code just continues to do what they used to do. 😄 

However, I just took a closer look into how HTTP requests are sent both in this class and the WP.com rest API one. I think it's impossible to be in a situation where this API client instance is deallocated whilst there are ongoing session tasks. Because [`self` is retained in API requests' completion handler](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/ea2d47e6a9723a2325cfd9d41c36357893ba0f63/WordPressKit/WordPressOrgXMLRPCApi.swift#L184).

Reply:
👌